apps/test/unit/lib/levelbuilder/code-docs-editor/OrderableListTest.jsx

```diff
@@ -65,9 +68,19 @@ describe('OrderableList', () => {
       .at(0)
       .simulate('click');
     expect(setListSpy).to.be.calledOnce.and.calledWith([
-      {key: 2},
-      {key: 1},
-      {key: 3}
+      {key: '2'},
+      {key: '1'},
+      {key: '3'}
     ]);
   });
+
+  it('hides delete button if item isnt deletable', () => {
+    const wrapper = shallow(
+      <OrderableList
+        {...defaultProps}
+        checkItemDeletionAllowed={item => item.key === '3'}
+      />
+    );
+    expect(wrapper.find('.fa-trash').length).to.equal(1);
```

Comment:
It looks like there should be 2 trash icons because there should be 3 items. Is that right?

Reply:
Only the third item is deletable. I waffled on which was to frame the function, i.e. `checkItemDeletionAllowed` or `preventItemDeletion`. Do you think the second one is clearer?